sdk/devcenter/developer-devcenter-rest/recordings/node/devcenter_dev_boxes_operations_test/recording_create_dev_box.json

```diff
@@ -1,49 +1,44 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://88888888-8888-8888-8888-888888888888-sdk-test-devcenter.devcenter.azure.com/projects/sdk-default-project/users/me/devboxes/SdkTestDevBox?api-version=2022-03-01-preview",
+      "RequestUri": "https://88888888-8888-8888-8888-888888888888-sdk-dc-na4b3zkj5hmeo.eastus.devcenter.azure.com/////////////////////////projects/sdk-default-project/users/me/devboxes/SdkTestDevBox?api-version=2022-11-11-preview",
```

Comment:
interesting to see so many `/`s. /cc @HarshaNalluru @timovv

Reply:
sanitizers/configs on the recorder look ok to me.

Maybe something messed up at the test-proxy level?

Reply:
@scbedd any hints?

Reply:
I'm not bothered by this in this PR since the build is green.
Logging an issue instead so Scott can check when he gets time.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/5296

Reply:
Scott made some changes to the logic not so long ago so that consecutive slashed would be preserved. Previously when there were multiple slashes at the start of the path they would be reduced to one slash, but we determined that to be incorrect behaviour. I suspect it might be the SDK that is adding the slashes and the test proxy is now preserving them when it wasn’t before.

Reply:
For now, I'll manually remove the extra slashes.

Reply:
@sebrenna I would recommend not touching recordings as in this case it is not blocking.

Reply:
@jeremymeng fair point - I thought that we were still blocked by [this issue](https://github.com/Azure/azure-sdk-for-js/issues/23420), and that therefore the green status on recorded tests weren't an indication the recordings were actually working. However, I see that we actually DO have the up-to-date test recorder in our project, so I'm just completely re-recording the tests now and won't touch the recordings after that.

Reply:
Sounds good @sebrenna